server/src/main/java/org/cloudfoundry/identity/uaa/oauth/jwt/JwtClientAuthentication.java

```diff
@@ -0,0 +1,62 @@
+package org.cloudfoundry.identity.uaa.oauth.jwt;
```

Comment:
Why no unit test coverage for this class from any of the tests in the server module?

Reply:
created test